packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -227,8 +235,10 @@ class DDSTabsExtended extends MediaQueryMixin(
       _tabItems.forEach((tab, index) => {
         (tab as DDSTab).setIndex(index);
 
-        if (!_isLargeOrGreater) {
+        //Checking for the presence of 'dds-processed' attr. Since the logic will only succeed once, attaching only one version of the 'click' event at a time when switching views.
+        if (!tab.hasAttribute('dds-processed') && !_isLargeOrGreater) {
           tab.addEventListener('click', this._handleAccordionClick.bind(this));
+          tab.setAttribute('dds-processed', 'true');
```

Comment:
Two nits:
1. Use `ddsPrefix` variable.
2. Since we're using this as a boolean attribute, it's a little cleaner to set the attribute to `''`. See https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute. My mistake for recommending the `true` value before!